CuratedContent/Jupyter4Science.md

```diff
@@ -0,0 +1,48 @@
+# Jupyter4Science: Better Practices for using Jupyter Notebooks for Science
+
+<!--deck text start-->
+[Jupyter4Science](https://jupyter4science.github.io/) is a **knowledge base** of original content and curated resources about developing and sharing Jupyter notebooks as they are used in the context of scientific applications.
+<!--deck text end-->
+
+#### Contributed by [Nicole Brewer](https://github.com/nicole-brewer)
+
+#### Publication date: March 31, 2023
+
+Resource information | Details 
+:--- | :--- 
+Resource name | Accessible Data and Code
```

Comment:
It seems like the Resource name should be "Jupyter4Science" rather than "Accessible Data and Code".  Are you okay with this change?

Reply:
I'm going to change the resource name to Jupyter4Science.  Hope that's okay.